chatFrontend/src/hooks/useGetMessages.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import useConversation from "../zustand/useConversation";
```

Comment:
Why use `zustand` package for state management while in most place you are already using contextAPI, can't that be utilized here?